app/middlewares/jwtMiddleware.go

```diff
@@ -0,0 +1,46 @@
+package middlewares
+
+import (
+	"star-pos/app/configs"
+	"strings"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+	echojwt "github.com/labstack/echo-jwt/v4"
+	"github.com/labstack/echo/v4"
+)
+
+func JWTMiddleware() echo.MiddlewareFunc {
+	return echojwt.WithConfig(echojwt.Config{
+		SigningKey:    []byte(configs.JWT_SECRET),
+		SigningMethod: "HS256",
+	})
+}
+
+func CreateToken(userId string) (string, error) {
+	claims := jwt.MapClaims{}
+	claims["authorized"] = true
```

Comment:
all jwt will always authorized. this claim will always true, which means we don't need this. please remove